cmd/maya-apiserver/app/server/snapshot_endpoint_v1alpha1.go

```diff
@@ -0,0 +1,189 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"github.com/openebs/maya/pkg/snapshot"
+)
+
+type snapshotAPIOpsV1alpha1 struct {
```

Comment:
Let us not have V1alpha1.
You may want to move this file to app/server/v1alpha1 instead

Reply:
I will do this as a part of another PR where I rearrange the dirs.